libclc/generic/libspirv/atomic/atomic_load.cl

```diff
@@ -43,6 +42,21 @@ IMPL_AS(long, l, , 8)
 IMPL_AS(unsigned long, m, u, 8)
 #endif
 
+#ifdef _CLC_GENERIC_AS_SUPPORTED
```

Comment:
This should be `#if` - it's always defined (assuming the header is included) but is either 1 or 0